### DISCUSSION TITLE
expanding the openai.playground functionality

### DISCUSSION THREAD

**Discussion by gotmike** [betalgo/openai#85] [Ideas]
I have been working on adding a few demos in the OpenAI.Playground project.

If you are accepting additional Contributors, please LMK if there are any guidelines.

I'd like to help out where possible.

Here is what I've done thus far...
1. Adding the ability to provide inputs to some of the test helpers using command-line arguments
2. Created a "combo completion" test helper that takes a prompt, runs through GPT completion, then takes the entire prompt + completion and sends to DALL-E with an image count, then generates the stated number of images. Afterwards, creates an HTML file with both prompts and the images, and opens in user's default browser. Not fancy, yet but does demonstrate the functionality in a little more user-friendly way
3. Made a couple small edits to Readme.md -- typo and clarification

**Comment by kayhantolga** [comment]
Yes please, I am more than happy with additional contributions. Unfourtanetly I do not have any guidelines at the moment but we can create a new one If you would like to help. 
The playground should stay with simple test methods. That way it would be simple to learn for beginners also I can test functionality easily after the changes.
1. Could you provide a sample please so I can understand better?
2. I was thinking of building the wiki page and also creating a page for showcases and samples. I think your project would fit there perfectly. 
3. What are these edits? Please create a pr or draft pr, and we can discuss it there.

**Comment by gotmike** [comment]
i created a pr this AM --> https://github.com/betalgo/openai/pull/87

**Comment by gotmike** [comment]
this pr includes all of the above
